braindecode/models/eegitnet.py

```diff
@@ -0,0 +1,313 @@
+# Authors: Ghaith Bouallegue <ghaithbouallegue@gmail.com>
+#
+# License: BSD-3
+import torch
+from torch import nn
+
+
+class DepthwiseConv2d(torch.nn.Conv2d):
+    def __init__(
+            self,
+            in_channels,
+            depth_multiplier=2,
+            kernel_size=3,
+            stride=1,
+            padding=0,
+            dilation=1,
+            bias=True,
+            padding_mode="zeros",
+    ):
+        out_channels = in_channels * depth_multiplier
+        super().__init__(
+            in_channels=in_channels,
+            out_channels=out_channels,
+            kernel_size=kernel_size,
+            stride=stride,
+            padding=padding,
+            dilation=dilation,
+            groups=in_channels,
+            bias=bias,
+            padding_mode=padding_mode,
+        )
+
+
+class EEG_ITNET(nn.Module):
```

Comment:
Let's make `EEG_ITNet` inherit from `nn.Sequential`, so it works easier with regression problems for example. With the comments below it will work well.